k8s/helm/hypercode/templates/secret.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "hypercode.fullname" . }}-secrets
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "hypercode.labels" . | nindent 4 }}
+  annotations:
+    helm.sh/resource-policy: keep
+type: Opaque
+data:
+  api-key: {{ .Values.secrets.values.apiKey | b64enc | quote }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)